doc/content/source/kernels/ScaledReaction.md

```diff
@@ -1,20 +1,16 @@
 # ScaledReaction
```

Comment:
Scaled Reaction was a previous attempted to give this kernel a generic name (it is a self consuming reaction scaled by a given value). 

I am not opposed to renaming this kernel. What do you find odd about the name and do you have any suggestions on the renaming?

Reply:
The reason I thought it might be a bit odd of a name was in the interest of generality because the kernel alone is not related to a reaction so maybe something like `ScaledSource` might be more appropriate. But considering the context of this application I think the scaled reaction name is certainly more appropriate than I had originally considered. So I don't have any issue keeping the name as it is.